### PR TITLE
fixing bug in tributors that does not paginate github!

### PR DESCRIPTION
**Pull request by vsoch**
This was discovered in US-RSE - we need to paginated the contributors API endpoint!

I also ran black and updated the license years - sorry for the extra noise! I tested this locally with the US-RSE repo and it seems to fix the issue.

cc @yarikoptic 

Signed-off-by: vsoch <vsoch@users.noreply.github.com>

### CODE REVIEW

File: tributors/version.py [hunk 1]
```diff
@@ -1,14 +1,14 @@
...
+Copyright (C) 2020-2022 Vanessa Sochat.
 
 This Source Code Form is subject to the terms of the
 Mozilla Public License, v. 2.0. If a copy of the MPL was not distributed
 with this file, You can obtain one at http://mozilla.org/MPL/2.0/.
 
 """
 
-__version__ = "0.0.19"
+__version__ = "0.0.2"
```
Comment (yarikoptic) [inline tributors/version.py]: ???

File: tributors/client/__init__.py [hunk 2]
```diff
@@ -2,7 +2,7 @@
 
 """
 
-Copyright (C) 2020 Vanessa Sochat.
+Copyright (C) 2020-2022 Vanessa Sochat.
```
Comment (yarikoptic) [inline tributors/client/__init__.py:5]: Which change is the fix? Should've been a separate commit from overall blacking

File: tributors/version.py (hunk 1 above)
Comment (vsoch) [reply tributors/version.py]: What's wrong with my version? 20 is after 19, if we do .20 pypi turns it into .2

File: tributors/client/__init__.py (hunk 2 above)
Comment (vsoch) [reply tributors/client/__init__.py:5]: yeah sorry about that! Let me comment on it so you see.

File: tributors/main/github.py [hunk 3]
```diff
@@ -182,30 +173,41 @@ def get_repo(repo):
...
+            if not os.environ.get("GITHUB_TOKEN"):
+                message += " you should export GITHUB_TOKEN to increase your API limits"
+            sys.exit(message)
+        new_contributors = {x["login"]: x for x in response.json()}
+
+        # This is the signal for we don't have any more pages
+        if not new_contributors:
+            break
+        contributors.update(new_contributors)
+        page += 1
```
Comment (vsoch) [inline tributors/main/github.py:203]: @yarikoptic here it is! And then below we just return the contributors lookup.

Comment (vsoch) [reply tributors/main/github.py:203]: https://github.com/con/tributors/pull/62/files#diff-475c09d4a87c68c77013c44aef91efe9d97fab48d3e856c5445f89191c04284aR183-R203

Comment (vsoch) [reply tributors/main/github.py:203]: The rest is running black and updating the license year, sorry for the noise! I kind of did it on auto-pilot.

File: tributors/version.py (hunk 1 above)
Comment (yarikoptic) [reply tributors/version.py]: Pypi mustn't turn .20 into .2. why do you think it does?

Anyways, given that tributors is used in production, making it 0.1.0 makes  sense while at it

Comment (vsoch) [reply tributors/version.py]: all set! https://github.com/con/tributors/pull/62/commits/c92933ed4a63cf3b9a077a91af9b4e2d3969d9cf

Comment (vsoch) [reply tributors/version.py]: And you're right, but I had the wrong version. When I do `0.0.01` it changes to:

```
Creating tar archive
removing 'tributors-0.0.1' (and everything under it)
```
but `0.0.20` works so let's use that here instead - it's not really a major version bump for a bugfix.

File: .github/workflows/test-tributors.yml [hunk 4]
```diff
@@ -11,12 +11,13 @@ jobs:
       - uses: actions/checkout@v2
 
       - name: Setup black environment
-        run: conda create --quiet --name black black pyflakes
+        run: conda create --quiet --name black pyflakes
 
       - name: Check formatting with black
         run: |
-          export PATH="/usr/share/miniconda/bin:$PATH"
+          export PATH="/usr/share/miniconda/bin:$PATH"
```
Comment (yarikoptic) [inline .github/workflows/test-tributors.yml]:
```suggestion
          export PATH="/usr/share/miniconda/bin:$PATH"
```

File: tributors/utils/prompt.py [hunk 5]
```diff
@@ -31,12 +30,12 @@ def request_input():
...
 def choice_prompt(prompt, choices, choice_prefix=None, multiple=False):
     """Ask the user for a prompt, and only return when one of the requested
-       options is provided.
-       Parameters
-       ==========
-       prompt: the prompt to ask the user
-       choices: a list of choices that are valid.   
-       multiple: allow multiple responses (separated by spaces)
+    options is provided.
+    Parameters
```
Comment (yarikoptic) [inline tributors/utils/prompt.py:35]: Needs empty line AFAIK
```suggestion

    Parameters
```

**Comment by vsoch** [comment]
Ready to go @yarikoptic ?

**Comment by vsoch** [comment]
Fixed, works like a charm https://github.com/USRSE/usrse.github.io/pull/726 <3

**Pull request merged**